app/helpers.php

```diff
@@ -5,10 +5,30 @@
  * Time: 12:10 AM
  */
 
+
 if (! function_exists('is_active_route')) {
-    function is_active_route($routePrefix) {
+    function is_active_route($routePrefix)
+    {
         if (strpos(request()->route()->getName(), $routePrefix) !== false) {
             return 'class = active';
         }
     }
-}
\ No newline at end of file
+}
+
+
+function mkoo_flash($message, $message_color = '#736F6F')
```

Comment:
I am aware the notification pan I added uses a pop up instead of an alert and gives it a different look . I will leave you to checkout the look , if its not good enough I will revert to the bootstrap alert .

Reply:
Please let's stick to Way's flash. Less code to maintain. If you must autohide the alert, there's an option for that.